src/nectar/cli.py

```
@@ -1810,13 +1653,15 @@ def follower(account):
-        if "default_account" in hv.config:
-            account = [hv.config["default_account"]]
-    for a in account:
+    accounts = list(account or [])
+    if not accounts and "default_account" in hv.config:
+        accounts = [hv.config["default_account"]]
+    for a in accounts:
         a = Account(a, blockchain_instance=hv)
         print("\nFollowers statistics for @%s (please wait...)" % a.name)
         followers = a.get_followers(False)
+        if isinstance(followers, list) and not isinstance(followers, Accounts):
+            raise ValueError("Expected Accounts object when raw_name_list is False")
```

Ruff (0.14.8)

Prefer `TypeError` exception for invalid type

(TRY004)